src/FacultyDirectory/Controllers/UsersController.cs

```diff
@@ -35,6 +35,15 @@ public ActionResult Get()
             return Ok(tokenString);
         }
 
+        [HttpGet("name")]
+        public ActionResult Name() {
+            var currentUser = this.User;
+            var firstName = currentUser.FindFirst(ClaimTypes.GivenName).Value;
```

Comment:
try changing to `currentUser.FindFirst(ClaimTypes.GivenName)?.Value;` for both claims (add the question mark) because if the results come back null/missing then I think it'll error out.  Instead it's better to do this null coalescence check.  Or without the fancy syntax you could just check if your result is null before using it.